argo-workflows/structure.json

```diff
@@ -1,9 +1,10 @@
 {
     "items": [
       { "path": "getting-started" },
+      { "path": "install-using-helm" },
```

Comment:
Maybe add this one more towards the end, before API? I'm not sure if we necessarily want to implicitly recommend Helm installs?